README.md

```diff
@@ -1,75 +1,114 @@
+- [Authors](#authors)
+- [Key Features](#key-features)
+- [Future Features](#future-features)
+- [Contributing](#contributing)
+- [Show Your Support](#show-your-support)
+- [Acknowledgements](#acknowledgements)
+- [License](#license)
 
+## About the Project
 
-## Built With
+This project implements the backend of hello-rails. PostgresSQL was used for database.
 
-- Major languages
-- Frameworks
-- Technologies used
+### Built With
 
-## Live Demo (if available)
+Hello rails-back-end was built with:
 
-[Live Demo Link](https://livedemo.com)
+- Ruby On Rails
+- PostgreSQL
 
+### Key Features <a name="key-features"></a>
 
-## Getting Started
+- Uses postgres to create a table of greetings.
+- Implements the display of random greetings by the fron-end.
+
+### Future Features <a name="key-features"></a>
+
+- Increase the database to add other tables.
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+### Related repos
 
+`https://github.com/Loltolo-Lesapiti/hello-react-front-end` - Hello rails front-end
+
+## Getting Started
 
-To get a local copy up and running follow these simple example steps.
+To get a local copy up and running, follow these steps:
 
 ### Prerequisites
 
-### Setup
+To run this project, you'll need to install:
 
-### Install
+- Ruby 2.7.0 or later
+- Rails 6.0 or later
+- PostgreSQL
 
-### Usage
+### Installation
 
-### Run tests
+1. Clone this repository to your desired folder:
 
-### Deployment
+`git clone https://github.com/Loltolo-Lesapiti/-hello-rails-back-end.git`
 
+`cd hello-rails-app`
 
+2. Install the required gems:
 
-## Authors
+`bundle install`
 
-👤 **Author1**
+3. Set up the database:
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+`rails db:create`
 
-👤 **Author2**
+`rails db:migrate`
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### Usage
 
-## 🤝 Contributing
+To run the project, execute the following command:
 
-Contributions, issues, and feature requests are welcome!
+`./bin/dev`
 
-Feel free to check the [issues page](../../issues/).
+### Testing
 
-## Show your support
+To run tests, run the following command:
 
-Give a ⭐️ if you like this project!
+`bundle exec rspec`
 
-## Acknowledgments
+## Authors
+
+👤 **Petro Loltolo Lesapiti**
+
+- GitHub: [@petrolesapiti](https://github.com/Loltolo-Lesapiti)
+- LinkedIn: [@petrolesapiti](https://www.linkedin.com/in/petrolesapitiloltolo/)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome! Feel free to check the [issues page]()
+
```

Comment:
- [OPTIONAL] When I click the link to the `issues page` link that you added, I get this 404 `error`. I would recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

Reply:
@Loltolo-Lesapiti, Your front-end repo is not apporved yet !!